apps/lfx-one/src/app/modules/dashboards/components/project-staff-card/project-staff-card.component.html

```
@@ -1,74 +1,69 @@
 <!-- Copyright The Linux Foundation and each contributor to LFX. -->
 <!-- SPDX-License-Identifier: MIT -->
 
-<div
-  class="p-4 bg-white border border-gray-200 hover:border-blue-500 rounded-lg flex-shrink-0 h-full w-[calc(100vw-3rem)] md:w-80"
-  data-testid="project-staff-card-section">
-  <div class="flex flex-col gap-2 h-full">
-    <!-- Header: icon + title (matches lfx-metric-card layout) -->
-    <div class="flex items-center gap-2">
-      <i class="fa-light fa-users-gear w-4 h-4 text-gray-500 flex-shrink-0"></i>
-      <h5 class="text-sm font-medium flex-1">Project Staff</h5>
+@if (loading() || hasError() || hasAnyStaff()) {
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)